app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
```

Comment:
[OPTIONAL] Kindly add descriptive future features in your Future features section.